templates/templates/blockified/single-product.html

```diff
@@ -69,4 +69,4 @@ <h2 class="wp-block-heading">Related products</h2>
 </div>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer"} /-->
+<!-- wp:template-part {"slug":"footer"} /-->
```

Comment:
Not sure if it's really important, but there used to be a white line at the end of this line.